components/openindiana/nvidia/Makefile

```diff
@@ -59,7 +60,7 @@ $(INSTALL_$(MK_BITS)):	$(BUILD_$(MK_BITS))
 	$(CP) -a $(SOURCE_DIR)/NVDAgraphics/reloc $(PROTO_DIR)/usr
 	$(TOUCH) $@
 
```

Comment:
That would need to make it generated into the sample-manifest too because long term plan is to make both the sample-manifest and the real manifest to match (as much as possible).  Now it is simpler the same diff as generated for the sample-manifest just apply to the real manifest too.